ckanext/ytp/comments/listeners.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+import ckan.plugins.toolkit as tk
+import ckan.lib.search as search
```

Comment:
This is fine, but `from ckan.lib import search` is simpler and works better if we end up importing multiple modules from `ckan.lib`.

Reply:
unsure, was just duplicating the original, tho it seems tests don't pass :/